plugin/bpm.events

```diff
@@ -33,7 +33,7 @@ bash_insert_chpwd() { bash_insert_command CHPWD_COMMAND "$@"; }
 
 
 ## hooks for prompt change (PROMPT_COMMAND)
-PROMPT_COMMAND="-chpwd-detect"
+PROMPT_COMMAND="${PROMPT_COMMAND:+$PROMPT_COMMAND ;} -chpwd-detect"
```

Comment:
```suggestion
PROMPT_COMMAND="${PROMPT_COMMAND:+$PROMPT_COMMAND;}-chpwd-detect"
```

This eventually got in (6f4c8b1ecce3c77c637128df776129f478996ca2) but still want to merge this PR.